static/index.html

```diff
@@ -28,30 +28,53 @@
   </div>-->
   <div id="snackbar"></div>
   <div class="flex-container">
-        <div class="input-group">
-          <span class="input-group-btn">
-            <button id="recordButton" class="btn btn-secondary" type="button" style="height:100%;padding-left:30px;padding-right:30px"><i id="recordIcon" class="fa fa-microphone" style=""></i></button>
-          </span>
-          <input type="text" class="form-control" placeholder="What do you need help with?" id="query-text">
-          <div class="input-group-btn">
-
-          <button id="category-dropdown-button" type="button" class="btn btn-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style="height:100%;border-radius:3px">
-              Categories
-            </button>
-            <div id="category-dropdown" class="dropdown-menu dropdown-menu-right">
-              <a class="dropdown-item" href="#">Action</a>
-              <a class="dropdown-item" href="#">Another action</a>
-              <a class="dropdown-item" href="#">Something else here</a>
-            </div>
-          </div>
-          <button type="button" id="submit-search" class="btn btn-primary" style="display:none;"></button>
+    <div class="input-group">
+      <span class="input-group-btn">
+        <button id="recordButton" class="btn btn-secondary" type="button" style="height:100%;padding-left:30px;padding-right:30px">
+          <i id="recordIcon" class="fa fa-microphone" style=""></i>
+        </button>
+      </span>
+      <input type="text" class="form-control" placeholder="What do you need help with?" id="query-text">
+      <div class="input-group-btn">
+
+        <button id="category-dropdown-button" type="button" class="btn btn-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+          aria-expanded="false" style="height:100%;border-radius:3px">
+          Categories
+        </button>
+        <div id="category-dropdown" class="dropdown-menu dropdown-menu-right">
+          <a class="dropdown-item" href="#">Action</a>
+          <a class="dropdown-item" href="#">Another action</a>
+          <a class="dropdown-item" href="#">Something else here</a>
         </div>
+      </div>
+      <button type="button" id="submit-search" class="btn btn-primary" style="display:none;"></button>
+    </div>
   </div>
   <div class="watson-container" id="result-container" style="display:none;">
     <div class="row">
       <div class="col-12">
-        <div id="result">
+        <div id="result0" id="response"></div>
+        <div class="panel-group" id="accordion">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h4 class="panel-title">
```

Comment:
Can we add some css to make it more obvious that these are accordions? currently they just look like hyper links.